packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -342,7 +343,18 @@ export async function produceBlockBody<T extends BlockType>(
             this.metrics?.blockPayload.emptyPayloads.inc({prepType});
           }
 
-          if (ForkSeq[fork] >= ForkSeq.deneb) {
+          if (ForkSeq[fork] >= ForkSeq.fulu) {
```

Comment:
you don't need to add  this code segment, fork >=deneb will already do the code flow, only now validateBlobsAndKzgCommitments needs to be fork aware if it batch verifies the proofs as well

Reply:
Sounds good, I'll update `validateBlobsAndKzgCommitments` and remove this segment